project/Dependencies.scala

```diff
@@ -12,7 +12,7 @@ object Dependencies {
 
   val workbenchUtilV    = "0.5-4c7acd5"
   val workbenchModelV   = "0.13-6dc016b"
-  val workbenchGoogleV  = "0.19-084fa1b"
+  val workbenchGoogleV  = "0.21-2b3109d-SNAP"
```

Comment:
Update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/270 is merged